get_google_oauth.js

```diff
@@ -28,7 +27,10 @@ fs.readFile('google-credentials.json', (err, content) => {
 function authorize(credentials) {
   const {client_secret, client_id, redirect_uris} = credentials.installed;
   const oAuth2Client = new google.auth.OAuth2(
-      client_id, client_secret, redirect_uris[0] );
+    client_id,
+    client_secret,
+    redirect_uris[0],
+  );
```

Comment:
This sort of thing is good. I vastly prefer breaking out parameters line by line this way. Vertical space doesn't need to be conserved.